### PR TITLE
PLATUI-981: Include options in HTML report to show only errors, hide known issues.

### PR DESCRIPTION
**Pull request by ejayaraman**
HTML Report Showing Options

### CODE REVIEW

**Review by vivrichards600 — DISMISSED** [review]
Changes look good. 

Would also be keen to have the devs take a look just to ensure this implementation is ok and that the filters make sense also to them.

File: app/resources/assets/style.scss [hunk 1]
```diff
@@ -71,6 +71,34 @@ body {
   }
 }
 
+.button {
+  margin-right:10px;
+}
+
+.violations--hide-non-errors, .violations--hide-known-issues {
+  .violation--known-info {
```
Comment (matthewmascord) [inline app/resources/assets/style.scss:79]:
```suggestion
  .violation--known-info, .violation--known-warning {
```

File: app/resources/assets/style.scss [hunk 2]
```diff
@@ -71,6 +71,34 @@ body {
...
+    display: none
+  }
+  .violation--known-warning {
+    display: none
+  }
+}
+
+.violations--hide-non-errors {
+  .violation--info {
```
Comment (matthewmascord) [inline app/resources/assets/style.scss:88]:
```suggestion
  .violation--info, .violations--warning {
```

File: app/resources/assets/style.scss [hunk 3]
```diff
@@ -71,6 +71,34 @@ body {
...
+  margin-right:10px;
+}
+
+.violations--hide-non-errors, .violations--hide-known-issues {
+  .violation--known-info {
+    display: none
+  }
+  .violation--known-warning {
+    display: none
+  }
```
Comment (matthewmascord) [inline app/resources/assets/style.scss:84]:
```suggestion
```

File: app/resources/assets/style.scss [hunk 4]
```diff
@@ -71,6 +71,34 @@ body {
...
+}
+
+.violations--hide-non-errors {
+  .violation--info {
+    display: none
+  }
+  .violations--warning {
+    display: none
+  }
```
Comment (matthewmascord) [inline app/resources/assets/style.scss:93]:
```suggestion
```

**Review by matthewmascord — APPROVED** [review]

**Pull request merged**